client/app/states/vms/vms.state.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+  'use strict';
+
+  angular.module('app.states')
+    .run(appRun);
+
+  /** @ngInject */
+  function appRun(routerHelper) {
+    routerHelper.configureStates(getStates());
+  }
+
+  function getStates() {
+    return {
```

Comment:
it this is a parent state, it should redirect somewhere, a vm list or explorer or something